symengine/sets.h

```diff
@@ -0,0 +1,62 @@
+/**
+ *  \file sets.h
+ *  Polynomial Manipulation
+ *
+ **/
+#ifndef SYMENGINE_SETS_H
+#define SYMENGINE_SETS_H
+
+#include <symengine/basic.h>
+#include <symengine/functions.h>
+#include <symengine/number.h>
+#include <symengine/rational.h>
+#include <symengine/complex.h>
+
+
+namespace SymEngine {
+
+class Interval : public Basic  {
```

Comment:
But @isuruf 
In sympy:
```
In [3]: a = Interval(2,3)

In [4]: b = Interval(5,8)

In [5]: a+b
Out[5]: [2, 3] U [5, 8]

```
And, I think we use that notation of `addition` for fuzzy sets only.